pkg/resources/resources.go

```diff
@@ -491,7 +491,7 @@ const (
 	EventRateLimitAdmissionPlugin = "EventRateLimit"
 
 	// KubeVirtInfraSecretName is the name for the secret containing the kubeconfig of the kubevirt infra cluster.
-	KubeVirtInfraSecretName = "cloud-controller-manager-infra-kubeconfig"
+	KubeVirtInfraSecretName = "kubevirt-infra-kubeconfig"
```

Comment:
Is it safe to change the name now? A new secret will be created? What about the old one?

Reply:
you're right, better not change it now.